book/website/project-design/logical-frameworks-overview.md

```diff
@@ -0,0 +1,20 @@
+(pd-logical-frameworks)=
+
+# Logical Frameworks
+
+## Prerequisites
+
+| Prerequisite | Importance |
+| -------------|----------|
+| {ref}`pd-checklist` | Helpful |
+
+
+## Summary
+
+Logic models and logical frameworks are akin to the blueprints we use for planning and evaluating programmes aimed at social change. Picture them as the game plan for an initiative or organisation, outlining the goals and the means to achieve them. 
```

Comment:
A gentle reminder that our style guide recommends each new sentence on a new line https://the-turing-way.netlify.app/community-handbook/style#write-each-sentence-in-a-new-line-line-breaks